server/public/model/integration_action.go

```diff
@@ -361,21 +360,15 @@ func (r *OpenDialogRequest) IsValid() error {
 	if err != nil {
 		multiErr = multierror.Append(multiErr, err)
 	}
+
 	return multiErr.ErrorOrNil()
 }
 
 func (d *Dialog) IsValid() error {
 	var multiErr *multierror.Error
 
 	if d.Title == "" || len(d.Title) > DialogTitleMaxLength {
-		multiErr = multierror.Append(multiErr, errors.New("invalid dialog title"))
-	}
-
-	if d.IconURL != "" && !IsValidHTTPURL(d.IconURL) {
-		multiErr = multierror.Append(multiErr, errors.New("invalid icon url"))
-	}
```

Comment:
Why was this removed? `IconURL` must be a URL and can't be a plugin path.

Reply:
Done. Misunderstanding